src/v/pandaproxy/client/record_set.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "kafka/types.h"
+#include "model/fundamental.h"
+
+namespace pandaproxy::client {
+
+class record_set_view {
```

Comment:
how does a view *own* a buffer? 

can you add comments on the design space here.

Reply:
Fixed in #515